.github/workflows/release-patch-2-trigger.yml

```diff
@@ -10,6 +10,11 @@ on:
         description: 'The head ref of the merged hotfix PR to trigger the release for (e.g. hotfix/v1.2.3/cherry-pick-abc).'
         required: true
         type: 'string'
+      workflow_ref:
```

Comment:
Shouldn't we add  `pull-requests: write` to both release-patch-2-trigger.yml‎ and release-patch-3-release.yml‎